support-frontend/app/utils/CheckoutValidationRules.scala

```diff
@@ -102,6 +102,7 @@ object CheckoutValidationRules {
       case d: DigitalPack => DigitalPackValidation.passes(createSupportWorkersRequest, d)
       case p: Paper => PaperValidation.passes(createSupportWorkersRequest, p.fulfilmentOptions)
       case _: GuardianWeekly => GuardianWeeklyValidation.passes(createSupportWorkersRequest)
+      case _: TierThree => GuardianWeeklyValidation.passes(createSupportWorkersRequest) // TODO: is this right?
```

Comment:
I think this is - but might be worth leaving a note why it's GWValidation (basically it's the same fields).